api/src/main/java/se/fortnox/reactivewizard/CollectionOptions.java

```diff
@@ -1,14 +1,23 @@
 package se.fortnox.reactivewizard;
 
+import javax.ws.rs.QueryParam;
+
 public class CollectionOptions {
     public enum SortOrder {
         ASC,
         DESC
     }
 
+    @QueryParam("limit")
     private Integer   limit;
+
+    @QueryParam("offset")
     private Integer   offset;
+
+    @QueryParam("sortBy")
```

Comment:
Are these case senesitive? If so, this is not compatible with `CollectionOptionsResolver` we have internally which uses `sortby`.